monkeytypes/credentials/lm_hash.py

```diff
@@ -12,8 +12,13 @@ class LMHash(InfectionMonkeyBaseModel):
     def __hash__(self) -> int:
         return hash(self.lm_hash)
 
-    @validator("lm_hash")
+    @field_validator("lm_hash")
+    @classmethod
     def validate_hash_format(cls, lm_hash):
         if not re.match(ntlm_hash_regex, lm_hash.get_secret_value()):
             raise ValueError("Invalid LM hash provided")
         return lm_hash
+
+    @field_serializer("lm_hash", when_used="json")
+    def dump_secret(self, v):
+        return v.get_secret_value()
```

Comment:
Use `get_plaintext()` here as well.

Reply:
We shouldn't use `get_plaintext` here as here we know that is going to be exactly LMHash

Reply:
I think it saves us from exposing and reusing `get_secret_value()`. With `get_plaintext()`, if something changes, we only need to change it in one place. It also saves someone reading this code from having to figure out why it's different in one place. `get_plaintext()` is simple and self-explanatory.